lib/components/Input/base/index.tsx

```diff
@@ -18,8 +28,9 @@ export interface InputProps extends InputHTMLAttributes<HTMLInputElement> {
     status?: Status;
     disabled?: boolean;
     variant?: Variants;
-    leftStatusMessage?: string;
-    rightStatusMessage?: string;
+    message?: string;
+    characterCounter?: boolean;
```

Comment:
Maybe we can make the props a little bit clearer. 

**showCharacterCount:** Indicates whether to display the character count on the input.
**enableCharacterCount:** Enables the character count feature for the input.
**displayCharacterCount:** Controls the visibility of the character count on the input.

We can choose from this based on what is the real purpose of the prop.